plugin.xml

```diff
@@ -5,8 +5,9 @@
     <description>${project.description}</description>
     <author>Guus der Kinderen</author>
     <version>${project.version}</version>
-    <date>2022-04-06</date>
+    <date>2023-11-14</date>
```

Comment:
```suggestion
    <date>2024-11-14</date>
```

Reply:
#picardfacepalm

Reply:
I've now force-pushed the correction as part of the commit that introduced the typo.